src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -40,12 +55,28 @@ const SubscriberHistory = () => {
       <div className="dp-box-shadow col-md-8 col-sm-8" style={{ margin: '80px auto' }}>
         <div className="col-md-8 col-sm-8" style={{ padding: '30px' }}>
           Email Suscriptor
-          <div style={{ display: 'flex' }}>
-            <input type="text" value={state.subscriberSearch} onChange={handeChange} />
-            <button className="dp-button button-medium primary-grey" onClick={searchSubscriber}>
-              Buscar
-            </button>
-          </div>
+          <Formik initialValues={getFormInitialValues()} onSubmit={onSubmit}>
+            {({ submitForm, handleChange }) => (
+              <Form style={{ display: 'flex' }}>
```

Comment:
What's this style doing here? Is it temporary?

Reply:
yes Gus still apply the styles

Reply:
i removed know